### PR TITLE
US: Add WUVP-DT1 (Univision 65)

### PR DESCRIPTION
**Pull request by MapGuy11**
I am working on adding more channels specifically from the Philadelphia OTA market. This link may not work right away. I'll let you know when to test it.

Thanks!

### CODE REVIEW

**Comment by freearhey** [comment]
@MapGuy11 You can convert PR to "Draft" in which case everyone will know that the work is still in progress. https://github.blog/changelog/2020-04-08-convert-pull-request-to-draft/

**Comment by MapGuy11** [comment]
Yeah, sorry I didn't know it was a thing.

**Comment by freearhey** [comment]
@MapGuy11 np

**Comment by MapGuy11** [comment]
@freearhey if you had to choose do you like h264 or h265 for video streaming?

**Comment by freearhey** [comment]
@MapGuy11 I personally don't care, but as I know h256 puts more stress on older systems because of their lack of hardware support.

**Comment by LaneSh4d0w** [comment]
Resolved conflict on your "us.m3u", was due to dupe links being added to your patch.

**Comment by halfrun** [comment]
hi 
try   https://streams.the6tv.duckdns.org:2443/locals/Philadelphia/wuvp-dt3.m3u8  it work

**Comment by freearhey** [comment]
@MapGuy11 It is better to hide comments rather than delete them to make it easier to track repeated violations of the rules.

**Comment by LaneSh4d0w** [comment]
> 
> 
> @MapGuy11 It is better to hide comments rather than delete them to make it easier to track repeated violations of the rules.

From what I've tracked, on the mails I had, it was telling other users to test an alternative link of his server, some link that MapGuy did not confirm himself.

**Comment by freearhey** [comment]
@MapGuy11 Any updates?

**Comment by MapGuy11** [comment]
> @MapGuy11 Any updates?

No sorry have been out on vaca.

**Comment by freearhey** [comment]
@MapGuy11 Don't be sorry. Have a great vacation! 🏖

**Comment by MapGuy11** [comment]
All good now the link should be up!

File: channels/us.m3u [hunk 1]
```diff
@@ -3660,3 +3660,5 @@ https://rockentertainment-zoomoo-1.samsung.wurl.com/manifest/playlist.m3u8
 http://188.40.68.167/russia/disney/playlist.m3u8
 #EXTINF:-1 tvg-id="MeiGuoQieDaDianShi.us" tvg-name="美国切达电视" tvg-country="US" tvg-language="" tvg-logo="" group-title="",美国切达电视 (1080p)
 https://live.chdrstatic.com/cheddar/primary/4.m3u8
+#EXTINF:-1 tvg-id="WUVP.us" tvg-name="WUVP-DT1" tvg-country="US" tvg-language="US" tvg-logo="https://static.locastnet.org/logo/504/1613144123818_UNI.png" group-title="",Univision Philadelphia (WUVP)
```
Comment (freearhey) [inline channels/us.m3u:3663]: This stream is `[Geo-blocked]`, please add this label to the title.
image

**Review by freearhey — CHANGES_REQUESTED** [review]

File: channels/us.m3u (hunk 1 above)
Comment (MapGuy11) [reply channels/us.m3u:3663]: Fixed try again shouldn't be blocked now.

Comment (LaneSh4d0w) [reply channels/us.m3u:3663]: vlcsnap-2021-06-12-19h01m47s721

Works for me.

Comment (freearhey) [reply channels/us.m3u:3663]: Now it works!

**Review by freearhey — APPROVED** [review]

**Pull request merged**